.hound.yml

```diff
@@ -1,14 +1,14 @@
 scss:
-  config_file: .scss-style.yml
+  config_file: config/hound/.scss-style.yml
```

Comment:
No need to make this a hidden file if it's in config/hound folder. Same for the rest files.